modules/internal/Bytes.chpl

```diff
@@ -740,12 +851,18 @@ module Bytes {
     }
 
     /*
-      :arg chars: A :record:`bytes` containing each character to remove.
-                  Defaults to `" \\t\\r\\n"`.
+      Strips given set of leading and/or trailing characters.
+
+      :arg chars: Characters to remove.  Defaults to `b" \\t\\r\\n"`.
+      :type chars: :record:`bytes`
```

Comment:
I think such `:type` declarations are redundant when they are clearly visible from the function signature